js/decklist/decklist.js

```diff
@@ -24,10 +24,12 @@ var Decklist = {
       main: [
         /^\s*(\d+)\s+\[.*\]\s+(.+?)\s*$/,      // MWS, what an ugly format
         /^\s*(\d+)x*\s(.+?)\s*$/,              // MTGO deck format (4 Brainstorm) also TCG (4x Brainstorm)
+        /^\s*()?(.+?)\s*$/,                    // Single card
       ],
       side: [
         /^\s*SB:\s*(\d+)\s+\[.*\]\s(.+?)\s*$/, // MWS, what an ugly format
         /^\s*SB:\s+(\d+)\s(.+?)\s*$/,          // Sideboard lines begin with SB:
+        /^\s*()?(.+?)\s*$/,                  // Single card
```

Comment:
This needs a couple more spaces to line up with the comments on previous lines.
```suggestion
        /^\s*()?(.+?)\s*$/,                    // Single card
```